packages/components/src/navigable-container/test/menu.js

```diff
@@ -14,16 +14,15 @@ import { UP, DOWN, LEFT, RIGHT, SPACE } from '@wordpress/keycodes';
  */
 import { NavigableMenu } from '../menu';
 
-function simulateVisible( wrapper, selector ) {
-	const elements = wrapper.getDOMNode().querySelectorAll( selector );
+function simulateVisible( elements ) {
```

Comment:
Seems like if we're passing `document.querySelectorAll` results to this function in all cases now anyways, we could have kept this as it was with receiving the `selector`, even if we don't want `wrapper` anymore.

Reply:
We still will need a node and a selector. I can make it looks similar.